test/simple_recipes.hpp

```diff
@@ -122,6 +122,127 @@ class cable1d_recipe: public simple_recipe_base {
     std::vector<mc_cell> cells_;
 };
 
+class gap_recipe_0: public recipe {
```

Comment:
These recipes are only used in `test_mc_cell_group`, and aren't generic, so they'd be better placed with the unit test itself.

Reply:
Done